src/azure-cli/requirements.py3.windows.txt

```diff
@@ -33,7 +33,7 @@ azure-mgmt-consumption==2.0.0
 azure-mgmt-containerinstance==9.1.0
 azure-mgmt-containerregistry==10.0.0
 azure-mgmt-containerservice==20.2.0
-azure-mgmt-core==1.3.1
+azure-mgmt-core==1.3.2
```

Comment:
May I ask why do you need to upgrade the package version for `azure-mgmt-core` and `msrest` in this PR?

Reply:
The pipelines were failing without these changes. Our new recoverservicesbackup SDK has dependencies on these versions -

[](https://github.com/Azure/azure-sdk-for-python/blob/d8c47ab85b0f97e8d51769be1ff63e8faa940723/sdk/recoveryservices/azure-mgmt-recoveryservicesbackup/setup.py#L72-L74)

Reply:
`azure-mgmt-core==1.3.2` only upgrade dependency for `azure-core` which has been upgraded in azure-cli. It does not have new feature so there is no influence for azure-cli.

Reply:
Thank you for helping clarify this upgrade

In addition, I can see from the history of these packages that the impact of their upgrade should not be big:

For https://pypi.org/project/azure-mgmt-core/1.3.2/
```
Updated mindep about azure-core from 1.23.0 to 1.24.0
```
`1.24.0` is currently the dependent version of CLI

For https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/requirements.py3.Darwin.txt#L11

For https://pypi.org/project/msrest/
```
Declare correctly msrest as Python 3.6 and more only for clarity #251
```
Related issue: https://github.com/Azure/msrest-for-python/issues/251